### PR TITLE
666 redirect to new

### LINKED ISSUE #666: After creating new node / routePath, redirect to correct the url
* Now after creating new routePath, url is still routePath/new, it should be instead for example something like /routePath/0052B,2019-02-27T15:25:06,1/?routes[0]=0052B

* After creating node, user should be redirected to node/{nodeId}

FE: https://github.com/HSLdevcom/jore-map-ui/pull/763

BE: https://gitlab.hsl.fi/jore/jore-map-backend/merge_requests/39

How to implement:
Make backend to return the data of created node:
response = {
    nodeId: ...,
    ...
}
res.status(200).json(response);

### PR DESCRIPTION
**Pull request by kronken**
Closes #666 
THE NUMBER OF THE BEAST

Test by creating new link, node and routepath

### CODE REVIEW

File: src/components/sidebar/routePathView/RoutePathView.tsx [hunk 1]
```diff
@@ -169,9 +171,19 @@ class RoutePathView extends ViewFormBase<IRoutePathViewProps, IRoutePathViewStat
 
     private save = async () => {
         this.setState({ isLoading: true });
+        let redirectUrl: string | undefined;
         try {
             if (this.props.isNewRoutePath) {
-                await RoutePathService.createRoutePath(this.props.routePathStore!.routePath!);
+                const routePathKey =
```
Comment (Divinare) [inline src/components/sidebar/routePathView/RoutePathView.tsx]: routePathPrimaryKey

File: src/models/ILine.ts [hunk 2]
```diff
@@ -1,8 +1,15 @@
 import TransitType from '~/enums/transitType';
 import { ILineRoute } from '.';
 
-export default interface ILine {
-    transitType: TransitType;
+interface ILinePrimaryKey {
```
Comment (Divinare) [inline src/models/ILine.ts]: export interface ILinePrimaryKey

is how we have exported enums / interfaces before, if you look for example at networkStore

File: src/services/routePathService.ts [hunk 3]
```diff
@@ -29,8 +30,10 @@ class RoutePathService {
     }
 
     public static createRoutePath = async (routePath: IRoutePath) => {
-        await ApiClient.createObject(endpoints.ROUTEPATH, routePath);
+        const routePathKey =
```
Comment (Divinare) [inline src/services/routePathService.ts]: response? There is already "as IRoutePathPrimaryKey"

**Comment by Divinare** [comment]
Made a card: #768

**Comment by kronken** [comment]
Fixed routepath warning message. Created task for node creation bug

**Pull request merged**